### DISCUSSION TITLE
Change font family to something other than IBM Plex

### DISCUSSION THREAD

**Discussion by supermonkeybrainz** [carbon-design-system/carbon#7828] [Help] [answered]
Is it possible to use a different font family than IBM Plex?  I know there are styles in `_font-family.scss`, but I can't seem to figure out how to override.  Is there an example somewhere I can use?  Any help would be appreciated.  Thanks!

**Comment by joshblack** [comment, marked as answer]
Hi there, @supermonkeybrainz! 👋 

To customize the font families that are used, you can define the `$carbon--font-families` variable before importing the styles for Carbon. By default, we expect at least the `sans` key to be defined but you can also pass in keys like `mono` and `serif` 😄 

```scss
$carbon--font-families: (
  sans: Helvetica,
);

@import 'path-to-carbon';
```

Here's a quick CodeSandbox link that does this technique along with the CSS Reset to use Helvetica on the page: https://codesandbox.io/s/custom-font-family-w68xc?file=/src/styles.scss

Hope this helps!

**Reply by supermonkeybrainz** [reply]
Will give that a go, thanks so much!
